docs/elasticsearch.md

```diff
@@ -2,36 +2,68 @@
 
 ElasticSearch is an integral component of Altis, enabling enhanced search and relevancy as well as powering the analytics data query layer.
 
+## Available Versions
+
+Elasticsearch defaults to version 6.8 however you can change the version in your config:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"local-server": {
+					"elasticsearch": "7.10"
+				}
+			}
+		}
+	}
+}
+```
+
+The current available versions are:
+
+- 7.10
+- 6.8 (default)
+- 6.3
+
+You can also use the major version on its own to get the latest minor version, for example "6" will resolve to version "6.8".
```

Comment:
Should we note ARM is only compatible with 7 here?

Reply:
Yis

Reply:
```suggestion
You can also use the major version on its own to get the latest minor version, for example "6" will resolve to version "6.8".

**Note**: If your device has an ARM chip you must use Elasticsearch 7 or higher.
```